cirq/google/gate_sets.py

```diff
@@ -12,116 +12,163 @@
-            ]),
-        op_serializer.GateOpSerializer(gate_type=ops.ZPowGate,
-                                       serialized_gate_id='exp_z',
-                                       args=[
-                                           op_serializer.SerializingArg(
-                                               serialized_name='half_turns',
-                                               serialized_type=float,
-                                               gate_getter='exponent')
-                                       ]),
-        op_serializer.GateOpSerializer(
-            gate_type=ops.XPowGate,
-            serialized_gate_id='exp_w',
-            args=[
-                op_serializer.SerializingArg(serialized_name='axis_half_turns',
-                                             serialized_type=float,
-                                             gate_getter=lambda x: 0.0),
-                op_serializer.SerializingArg(serialized_name='half_turns',
-                                             serialized_type=float,
-                                             gate_getter='exponent'),
-            ]),
-        op_serializer.GateOpSerializer(
-            gate_type=ops.YPowGate,
-            serialized_gate_id='exp_w',
-            args=[
-                op_serializer.SerializingArg(serialized_name='axis_half_turns',
-                                             serialized_type=float,
-                                             gate_getter=lambda x: 0.5),
-                op_serializer.SerializingArg(serialized_name='half_turns',
-                                             serialized_type=float,
-                                             gate_getter='exponent'),
-            ]),
-        op_serializer.GateOpSerializer(gate_type=ops.CZPowGate,
-                                       serialized_gate_id='exp_11',
-                                       args=[
-                                           op_serializer.SerializingArg(
-                                               serialized_name='half_turns',
-                                               serialized_type=float,
-                                               gate_getter='exponent')
-                                       ]),
-        op_serializer.GateOpSerializer(
-            gate_type=ops.MeasurementGate,
-            serialized_gate_id='meas',
-            args=[
-                op_serializer.SerializingArg(
-                    serialized_name='key',
-                    serialized_type=str,
-                    gate_getter=protocols.measurement_key),
-                op_serializer.SerializingArg(serialized_name='invert_mask',
-                                             serialized_type=List[bool],
-                                             gate_getter='invert_mask')
-            ])
+        _SYCAMORE_SERIALIZER, *SINGLE_QUBIT_SERIALIZERS,
+        *SINGLE_QUBIT_HALF_PI_SERIALIZERS, MEASUREMENT_SERIALIZER
```

Comment:
nit: personally I'd add a comma on the end to make it format one per line.  But I have a tall screen :)

Reply:
Done.